### PR TITLE
Update Oregon.md

### PR DESCRIPTION
**Pull request by emarcey**

### CODE REVIEW

File: reports/Oregon.md [hunk 1]
```diff
@@ -98,3 +98,35 @@ id: or-portland-6
...
+
+tags: tear-gas, inhumane-treatment
+
+id: or-salem-1
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1268940700944605187
+* https://www.buzzfeednews.com/article/davidmack/salem-oregon-cop-warn-white-armed-men-playing-favorites
+* https://twitter.com/Satellit3Heart/status/1268863536299675648
```
[the remark this answers is not in the document]
Comment (emarcey) [reply reports/Oregon.md]: Sure, I can do that. But I'm still working under the change we made to consider "abuse of power" as valid, not strictly "brutality" - and I think this is clearly an abuse of power

**Comment by emarcey** [comment]
https://github.com/2020PB/police-brutality/pull/626

File: reports/Oregon.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (maxspencer) [reply reports/Oregon.md]: I'm inclined to agree with you, perhaps as #626 we can update the contribution guidelines to make them even more explicit about this - perhaps even pointing to that PR as an example?

[the remark this answers is not in the document]
Comment (emarcey) [reply reports/Oregon.md]: Sure I'll include a few examples of "abuse of power." 

I think this "playing favorites" is certainly one.

I'm inclined to consider the case where police harass a teen for his social media as an abuse of power as well:
https://github.com/2020PB/police-brutality/pull/611

And I had obviously included a few other cases in other PRs. I'm not sure where you stand on them personally vs. what the guidelines say

[the remark this answers is not in the document]
Comment (maxspencer) [reply reports/Oregon.md]: Yeah I think the key thing will be to tighten up the definition of "abuse of power".

**Pull request merged**